pkg/client/messagepickup/client.go

```diff
@@ -0,0 +1,73 @@
+/*
+Copyright Scoir Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package messagepickup
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/hyperledger/aries-framework-go/pkg/didcomm/common/service"
+	"github.com/hyperledger/aries-framework-go/pkg/didcomm/protocol/messagepickup"
+)
+
+type provider interface {
+	Service(id string) (interface{}, error)
+}
+
+// Client enable access to message pickup api
+type Client struct {
+	service.Event
+	messagepickupSvc protocolService
+}
+
+type protocolService interface {
```

Comment:
how will the outbound `noop` message be sent?

Reply:
@m00sey ^^

Reply:
@llorllale I'm literally wrapping up tests after reading https://github.com/hyperledger/aries-rfcs/issues/501 - I'll open another PR.